lib/kcl.js

```diff
@@ -221,7 +221,21 @@ module.exports = function(config, kinesis) {
 
       if (resp.data.Records && resp.data.Records.length > 0) {
```

Comment:
Adding a delay causes the test to sometimes hang. Per chat with @l-r, I should just leave this as `2500` for now.